### ISSUE TITLE
Mitmproxy out of file handles

### ISSUE THREAD

**Issue by mlandauer** [openaustralia/morph#328]
Need to set up a test harness that can reproduce the problem

**Comment by mlandauer** [comment]
It looks like there's some really helpful new information on the original issue ticket https://github.com/mitmproxy/mitmproxy/issues/383

**Comment by mlandauer** [comment]
One issue that I came across is that the previous iptables setup scripts were occasionally injecting the same iptables rules twice. I don't know if this is the source of the previous issue.

Anyway, assuming it was, I added two simple scripts that setup (and tear down) the iptables setup for mitmproxy so things can be tested without rebooting the morph server.

After upgrading mitmproxy to the latest version and adding the iptables rules everything looks to be working on the production server.

In just half an hour or so it's already logged about 2,500 web connections.

I've been monitoring the number of open file handles with
```
lsof -n | grep mitmdump | grep TCP | wc
```

And so far it's looking very stable. So fingers crossed!
